zer0-os/zOS
# Log in to Matrix via JWT

### What does this do?

Adds login to Matrix via JWT:
* Login happens as soon as you connect
* If you don't have a Matrix id for the user the JWT connection creates a Matrix account automatically. This then sets your matrix id locally to ensure things ar synced up.

### How do I test this?

Register as a new user in zOS. Verify you can create conversations with existing Matrix/zOS users and verify that other users can create conversations with your new user.
